chat-plugins/info.js

```diff
@@ -1899,10 +1901,10 @@ exports.commands = {
 		Rooms.SimulatorProcess.eval('Config.potd = \'' + toId(target) + '\'');
 		if (target) {
 			if (Rooms.lobby) Rooms.lobby.addRaw("<div class=\"broadcast-blue\"><b>The Pok&eacute;mon of the Day is now " + target + "!</b><br />This Pokemon will be guaranteed to show up in random battles.</div>");
-			this.logModCommand("The Pok\u00e9mon of the Day was changed to " + target + " by " + user.name + ".");
+			this.modlog('POTD', null, target);
```

Comment:
This should continue to modlog who changed it

Reply:
same as above